docs/user-guide/recipes.rst

```diff
@@ -356,56 +356,54 @@ both a JSON file for use with crane, and the static Image files to which crane
 will redirect requests. See the `Crane`_ documentation for how to use that
 tarball.
 
-Upload v2 schema 2 Images to Pulp
----------------------------------
 
-.. note::
-
-    As of the time of this writing, ``skopeo copy`` can only output Docker v2
-    schema 2 content. Thus, only Docker v2 schema 2 content can be uploaded
-    to Pulp for now. In order to get your own Docker v2 schema 1 content into
-    Pulp, it is possible to run your own Docker registry and point Pulp's
-    feed URL at it and synchronize.
+Upload v2 schema 2 and schema 1 Images to Pulp
+----------------------------------------------
 
 To upload a Docker Image to Pulp, first you must save its repository with Skopeo.
```

Comment:
Maybe, add a hyperlink to [Skopeo](https://github.com/projectatomic/skopeo) here, otherwise, LGTM :+1: